staker-rewards/README.adoc

```diff
@@ -1,35 +1,113 @@
+
+* If the operator does not have `BondedECDSAKeepFactory` contract or tBTC sortition
+  pool authorized at `T_start`, they are not getting any rewards for interval `T`.
+  
+* If the operator does not have a minimum KEEP stake at `T_start`, they are not
+  getting any rewards for interval `T`.
+  
+* If the operator has more `ETH_unbonded` than the minimum unbonded value and is
+  not in the sortition pool at `T_start`, they are not getting any rewards for
+  interval `T`. In other words, if the operator is eligible to be in the
+  sortition pool, they have to be in the sortition pool. Operators who are not
+  in the sortition pool because all of their ETH is bonded are still getting
+  rewards because that ETH is still under the system’s management.
+  
+* If the operator deauthorized tBTC sortition pool between `T_start` and `T_end`, 
+  they are not getting any rewards for interval `T`.
+  
+* If the operator ever committed a fraud, they are not getting any rewards for
+  interval `T`. 
+  
+* If the operator did not meet the signature SLA between `T_start` and `T_end`,
+  they are not getting any rewards for interval `T`. For less than 20
+  closed/terminated keeps, no more than one could be terminated with a signature
+  timeout. For 20 or more closed/terminated keeps no more than 5% could be
+  terminated with a signature timeout. Signature SLA does not include not
+  answered courtesy calls.
+  
+* If the operator did not meet the key generation SLA between `T_start` and
+  `T_end` , they are not getting any rewards for interval `T` . For less than
+  10 key generations, one of them can fail. For 10 or more key generations,
+  no more than 10% can fail.
+  
+=== The Formula
+
+image::rewards.png[staker rewards formula]
+
+== Generating Reward Distribution
+=== Prerequisites
 
 To run the script, make sure you have the following prerequisites:
 
-- Node.js (at least v14.3.0). We recommend to use
+* Node.js (at least v14.3.0). We recommend to use
   the https://github.com/nvm-sh/nvm[Node Version Manager] to get the right
   Node.js version with no troubles. You can install this tool by running
   `brew install nvm` on macOS or follow the https://github.com/nvm-sh/nvm#installing-and-updating[installation guide]
-  for other operating systems.
+  for other operating systems. To make sure you use the right Node.js version,
+  you can run `nvm use 14.3.0`.
 
-== How to run
+=== How to run
 
-1. Make sure you use the right Node.js version:
-+
+To verify operator rewards and display summary in a table, run:
 ```
-nvm use 14.3.0
+./rewards-merkle-generator.sh -v -h <eth_host> -s <start_timestamp> -e <end_timestamp> -r <reward_allocation>
 ```
-2. Install the NPM dependencies if not done yet:
-+
-```
-npm install
-```
-3. Run the script along with environment variables:
-+
+
+This command runs all calculations against mainnet data and displays operator 
+rewards in a human-friendly way in a table with numbers rounded to two decimal
+places.
```

Comment:
What do you think about adding a note that upon claiming rewards, KEEP will be transferred to beneficiary and not the operator?